.github/workflows/links.yml

```diff
@@ -0,0 +1,27 @@
+name: Links
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 18 * * *"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1.0.9
+        with:
+          args: --verbose --no-progress **/*.md **/*.html
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        
+      - name: Create Issue From File
```

Comment:
I don't think we need to create an issue as if the Workflow fails, we'd just have the PR branch updated.

Reply:
Oh okay, I will update the PR

Reply:
Just to be sure, we don't need the environment variable too right?

Reply:
I don't think so.